test/functional/ui/output_spec.lua

```diff
@@ -230,4 +232,20 @@ describe("shell command :!", function()
       ]])
     end)
   end)
+
+  if iswin() or eval('executable("pwsh")') then
+    it('powershell supports literal strings', function()
+      set_shell_powershell()
+      local screen = Screen.new(30, 5)
+      screen:attach()
+      feed_command([[!$a = 1; 'echo $a']])
+      retry(nil, 2000, function()
+        screen:expect{any='echo %$a', timeout=1000}
+      end)
+      feed_command([[!$a = 1; "echo $a"]])
+      retry(nil, 2000, function()
+        screen:expect{any='echo 1', timeout=1000}
+      end)
```

Comment:
screen:expect already retries , so wrapping it with retry() is discouraged.

Reply:
This however results in shorter timeouts (if that was the intention) I guess.